Frontend/voitheia-App/src/app/components/layout/footer/footer.component.scss

```diff
@@ -2,17 +2,19 @@
 
 :host {
   .page-footer {
-    position: absolute;
+    display: block;
```

Comment:
Are you sure that the footer will always be at the bottom (even when the page is not using all the space) with this configuration?

Reply:
Yes, footer will always be at the end of all content with some small distance inbetween. It will also always start earliest below the viewport-height no matter how high the content is (which I find quite nice look and feel wise)